rpc/src/module/chain.rs

```diff
@@ -1193,6 +1194,46 @@ pub trait ChainRpc {
     /// ```
     #[rpc(name = "get_consensus")]
     fn get_consensus(&self) -> Result<Consensus>;
+
+    /// Returns the past median time by block hash.
+    ///
+    /// ## Params
+    ///
+    /// * `block_hash` - the highest block hash
```

Comment:
I think `the highest block hash` may create confusion about this param should be the tip block hash or block hash of any height in the chain

Reply:
Thanks. I agree with that. What about the below, or any suggestions?

> "A median time is calculated for a consecutive block sequence. `block_hash` indicates the highest block of the sequence."

Reply:
LGTM

Reply:
Updated document.